authentication/account/worker/user_deactivation_worker.go

```diff
@@ -53,11 +55,21 @@ func (w *userDeactivationWorker) deactivateUsers() {
 		}, "error while notifying users to deactivate")
 		return
 	}
+
+	rescheduledDeactivation := time.Now().Add(w.rescheduleDelay)
```

Comment:
can be the same time for all accounts in this batch, or we could compute a new scheduled time for each account. But I guess that does not make much difference.